auth/src/main/java/com/firebase/ui/auth/ui/FragmentHelper.java

```diff
@@ -0,0 +1,31 @@
+package com.firebase.ui.auth.ui;
```

Comment:
It looks like we have two copies of this class, one in the `.auth.ui` package and one directly in `.auth`.  I imagine you meant to delete one?